### PR TITLE
ACT-2086 | Add removeDataSource to policies resource

### PR DESCRIPTION
**Pull request by mmitoff**

### CODE REVIEW

**Comment by mmitoff** [comment]
Review pending approval on approach in vnext PR

**Review by jpartin2 — APPROVED** [review]

File: src/sdk/resources/policies.ts [hunk 1]
```diff
@@ -29,8 +29,12 @@ export class PoliciesResource extends PagedResource<Policy> {
 		return getTypedResponse<EntityTypeAssignment>(result);
 	}
 
-	async deleteEntityAssignment(id: string, entityTypeId: string, token?: CancellationToken): Promise<boolean> {
-		return await this.httpClient.delete(`${this.resourcePath}/${id}/entity_types/${entityTypeId}`, null, token);
+	deleteEntityAssignment(id: string, entityTypeId: string, token?: CancellationToken): Promise<boolean> {
+		return this.httpClient.delete(`${this.resourcePath}/${id}/entity_types/${entityTypeId}`, null, token);
```
Comment (MicheleBuselli) [inline src/sdk/resources/policies.ts:33]: Why did you remove await?

Comment (mmitoff) [reply src/sdk/resources/policies.ts:33]: Because we're not using the value of the promise here

**Review by jpartin2 — APPROVED** [review]
